Gemfile

```diff
@@ -24,7 +24,7 @@ gem "curb", "~> 0.8"
 
 gem "nokogiri", "~> 1.6"
 
-gem "rails_12factor"
+gem "rails_12factor", group: :production
```

Comment:
We would then need to run staging, in ENV production, which is fine, but need to differentiate by perhaps domain (as we do in other apps)